syntax = "proto3";

package tasklet_tests;

import "tasklet/api/tasklet.proto";

message Entry {
  string value = 1;
}

message PyEntry {
  option (tasklet.py_adapter) = "tasklet.runtime.tests.common:DummyPyAdapter";
  string value = 1;
}

message Context1 {
  Entry a = 1;
  Entry b = 2;
  tasklet.ServiceRef ref = 3;
}

message Context2 {
  Entry a = 1;
  PyEntry b = 2;
}
